WcaOnRails/spec/spec_helper.rb

```diff
@@ -84,4 +84,15 @@
   # test failures related to randomization by passing the same `--seed` value
   # as the one that triggered the failure.
   Kernel.srand config.seed
+
+  config.before(:suite) do
+    %w(Countries Continents Events Rounds Formats teams).each do |table|
+      # -- Uncomment to regenerate --
+      # ActiveRecord::Base.connection.execute("TRUNCATE #{table};")
+
+      if ActiveRecord::Base.connection.execute("SELECT COUNT(*) FROM #{table}").first == [0]
+        load "#{Rails.root}/db/seeds/#{table.underscore}.seeds.rb"
```

Comment:
Unfortunately we cannot use the code you've presented because the model [`Country`](https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/models/country.rb) isn't connected with Countries table (the same goes for all except teams). I remember that Jeremy made the models *virtual* in this way.